tests/testthat/test-verb-summarise.R

```diff
@@ -20,7 +20,7 @@ test_that("can't refer to freshly created variables", {
   mf1 <- lazy_frame(x = 1)
   expect_error(
     summarise(mf1, y = sum(x), z = sum(y)),
```

Comment:
Can you switch this to a snapshot test? I think reviewing the full generated error is useful in these cases.

Reply:
👍